HACKING.md

```diff
@@ -35,6 +35,28 @@ Follow the instructions to install the operator, e.g.:
 $ oc apply -f /tmp/manifests/path
 ```
 
+#### Building & Running the Operator Locally
+
+This allows you to quickly test changes to the operator without pushing any code or images to the cluster.
+
+To build the operator binary locally:
+
+```
+$ make build
+```
+
+To run the operator binary in the cluster from your local machine (as opposed to on the cluster in a pod):
+
+```
+$ make run-local
+```
+
+Note, to re-scale the operator on the cluster after local testing is complete, run:
+
+```
+$ oc scale --replicas 1 -n openshift-ingress-operator deployments ingress-operator
```

Comment:
`make run-local` also scales CVO to 0 replicas; should we suggest `oc scale --replicas 1 -n openshift-cluster-version deployments/cluster-version-operator` too, or recommend it instead (once it is running, CVO will scale the ingress operator up too)?

Reply:
I think providing only the command to scale the CVO back up would be sufficient. On that note, should the `Building Locally & Deploying to the Cluster` section provide a similar suggestion for scaling the CVO, since `make uninstall` scales the CVO to 0 replicas as well?

Reply:
> On that note, should the `Building Locally & Deploying to the Cluster` section provide a similar suggestion for scaling the CVO, since `make uninstall` scales the CVO to 0 replicas as well?

Yeah, that would be reasonable.

Reply:
Fixed, thanks!!